# 14 Positioning z-score

Does the gap between five-print RWTC returns when the managed-money z-score is above 1.5, and when it is not, keep its sign after 2019?

Holdout starts 2020-01-01. Verdict: failed.

Managed-money net divided by open interest on CFTC 067651. Z-score uses the train mean and the train population standard deviation of that ratio. Train mean 0.1146. Train standard deviation 0.0386. The percentile is the share of train ratios at or below the current print. Spreads are not inside the net.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'cot-z')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
Train gap, z above 1.5 minus the rest, is -0.128% (39 extreme, 480 rest). Holdout gap is 1.722% (11 extreme, 340 rest). The holdout flipped the train sign.
Five-print RWTC return when COT_Z is above 1.5
Train | 39 | 480 | -0.128%
Holdout | 11 | 340 | 1.722%


## Method

The report date is joined to RWTC on that date, or to the next print within 3 calendar days.

## Result

Train gap, z above 1.5 minus the rest, is -0.128% (39 extreme, 480 rest). Holdout gap is 1.722% (11 extreme, 340 rest). The holdout flipped the train sign.